alpha/website/js/dataset2/obj-exchange.js

```diff
@@ -403,7 +403,7 @@ this.raw_json = j;
 
         <div class="col-8">
             <div class="row">
-                <span class="font-weight-bold font-heavy"><br><h6>  ${html_name} </h6></span>
+                <span class="font-weight-bold font-heavy"><br><h6>  ${html_name} </h6><br></span>
```

Comment:
try col-12

Reply:
after js cache updating mb-2 w/top as well: mt-2 did not give the same result; not sure how col helps w/vertical spacing

Reply:
after js cache, now seeing some change w/mb-2 and mt-2, not same, but assume ok?